### PR TITLE
Optional sigmoid transformation and bugfix

### PR DESCRIPTION
**Pull request by bkleyn**

### CODE REVIEW

File: mab2rec/rec.py [hunk 1]
```diff
@@ -288,7 +288,7 @@ def predict_expectations(self, contexts: Union[None, List[List[Num]], np.ndarray
         return self.mab.predict_expectations(contexts)
 
     def recommend(self, contexts: Union[None, List[List[Num]], np.ndarray, pd.Series, pd.DataFrame] = None,
-                  excluded_arms: List[List[Arm]] = None, return_scores: bool = False) \
+                  excluded_arms: List[List[Arm]] = None, return_scores: bool = False, transform_scores: bool = True) \
```
Comment (skadio) [inline mab2rec/rec.py:291]: can you remind me again why this was useful to have this option? 

re: naming, given that this is a boolean.. is_sigmoid? has_sigmoid? has_sigmoid_transform? is_sigmoid_applied?

Comment (skadio) [reply mab2rec/rec.py:291]: is this a "breaking" change now, OR we are okay since it has a default setup as True

**Review by skadio — APPROVED** [review]

File: mab2rec/rec.py (hunk 1 above)
Comment (bkleyn) [reply mab2rec/rec.py:291]: No, not breaking. Optional argument and default behavior is the unchanged.

Comment (bkleyn) [reply mab2rec/rec.py:291]: re: naming - how about "apply_sigmoid"?

Comment (skadio) [reply mab2rec/rec.py:291]: @wddcheng when you get a chance, can you change this to apply_sigmoid, merge the PR, and close the related Issues please?

Comment (wddcheng) [reply mab2rec/rec.py:291]: will do!

**Pull request merged**